### DISCUSSION TITLE
Image Countdown

### DISCUSSION THREAD

**Discussion by zicklag** [YassinEldeeb/deadliner#10] [Ideas]
I just showed this app to my brothers and both of them mentioned that you should be able to set different images for the desktop background that could, for instance, increase in "intensity" as the deadline gets nearer. I showed them this background and they were like "I hope the 3 months left image is much calmer!" :)

![image](https://user-images.githubusercontent.com/25393315/171746387-6ee1086c-c2e4-4fdb-8e65-a3f198100d12.png)

I'm not sure what the best UX would be for setting the different images yet, but I figured I'd post the idea.

**Comment by YassinEldeeb** [comment]
Really really dope, I also see it as a neat feature.

But as you've mentioned I don't really have an idea on how that would be implemented in a nice user interface like how can a user set different wallpapers based on different dates thresholds, etc...

If you can please open an issue with this feature request so that we can get some ideas on how it would be implemented from other devs, I'd really appreciate it 🙏

**Reply by YassinEldeeb** [reply]
The way I think if implementing it rn is a predefined wallpapers with predefined date thresholds, but unfortunately not all users would agree on these thresholds nor the defined wallpapers.
